.github/workflows/ccruntime_e2e.yaml

```diff
@@ -25,6 +25,7 @@ jobs:
           sudo apt-get install -y ansible python-is-python3
 
       - name: Run e2e tests
+        timeout-minutes: 30
```

Comment:
I'd suggest at least 45m (60m to be pretty sure) since the runs in this PR took up to 26m.

Reply:
Updated to 45, thanks!